Libraries/WebRTC/webrtcSetup.ps1

```diff
@@ -71,11 +71,22 @@ CMD /C "git clean -f"
 CMD /C "git checkout -b patch_branch refs/remotes/branch-heads/58"
 CMD /C "gclient sync --jobs 16"
 CMD /C ("git apply --ignore-whitespace " + $PSScriptRoot + "\nvencoder.patch")
+CMD /C ("git add --all")
 CMD /C 'git commit -am "nvencoder patch"'
 CMD /C ("git apply --ignore-whitespace " + $PSScriptRoot + "\videoframemetadata.patch")
+CMD /C ("git add --all")
 CMD /C 'git commit -am "videoframemetadata patch"'
 CMD /C ("git apply --ignore-whitespace " + $PSScriptRoot + "\nvencodernobuffer.patch")
+CMD /C ("git add --all")
 CMD /C 'git commit -am "nvencodernobuffer patch"'
+CMD /C ("git apply --ignore-whitespace " + $PSScriptRoot + "\cuda.patch")
+CMD /C ("git add --all")
+CMD /C 'git commit -am "cuda patch"'
+Set-Location "third_party"

```

Comment:
this differs from the flow for applying other patches, is there a reason we need to be in /third_party to apply this one?

Reply:
this is different because we moved the nvencode library into third_party. This is in line with Google's best practices in adding a new library to WebRTC and it keeps it separated from the encoder implementation